baremetal/metal3machine_manager.go

```diff
@@ -609,15 +609,15 @@ func (m *MachineManager) Delete(ctx context.Context) error {
 						}
 						// Check if machine is ControlPlane
 						if m.isControlPlane() {
```

Comment:
this I double-checked and it uses [this](https://github.com/kubernetes-sigs/cluster-api/blob/e55c6bfad66bb7c39d95037b7334c795e6201b23/util/util.go#L133) and should be fine as is

Reply:
Thanks!